pkg/api/api_test.go

```diff
@@ -19,13 +19,13 @@
 package api
 
 import (
-	"github.com/dynatrace/dynatrace-configuration-as-code/pkg/environment/v1"
+	"github.com/dynatrace/dynatrace-configuration-as-code/pkg/converter/v1environment"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 )
 
-var testDevEnvironment = v1.NewEnvironment("development", "Dev", "", "https://url/to/dev/environment", "DEV")
+var testDevEnvironment = v1environment.NewEnvironmentV1("development", "Dev", "", "https://url/to/dev/environment", "DEV")
```

Comment:
stupid question, why does the current api_test need a v1 environment?

Reply:
okay, I just looked... it is used twice for: `.GetUrl(testDevEnvironment.GetEnvironmentUrl()),` :sweat_smile: 
You could replace this with a string